docs/all_the_issues.md

```diff
@@ -26,4 +26,18 @@ Then you can start your database with:
 sudo docker run -d -e REDIS_ARGS="--requirepass QzmCUD3C3RdsR" --name redis-stack -p 6379:6379 -p 8001:8001 -v /home/ubuntu/redis-data/:/data/ redis/redis-stack:latest
 ```
 
-Redis saves snapshots of the database every few minutes. You can find them in the corresponding folder (for example, `\data\dump.rdb`). Use the `sudo docker cp <container_id>:/data/dump.rdb /tmp` to obtain the snapshot from the container (in this case, the copied data is stored in `/tmp/snap-private-tmp/snap.docker/tmp`).
+Redis saves snapshots of the database every few minutes. You can find them in the corresponding folder (for example, `./data/dump.rdb`). Use the `sudo docker cp <container_id>:/data/dump.rdb /tmp` to obtain the snapshot from the container (in this case, the copied data is stored in `/tmp/snap-private-tmp/snap.docker/tmp`).
+
+When utilizing the database in python code (as introduced in [here](https://github.com/sotopia-lab/sotopia/blob/main/notebooks/redis_stats.ipynb)), make sure you use in the command line:
+
+```sh
+export REDIS_OM_URL="redis://:QzmCUD3C3RdsR@localhost:6379"
+```
+
+instead of adding variables in the python code
+
+```sh
+os.environ['REDIS_OM_URL'] = "redis://:QzmCUD3C3RdsR@localhost:6379"
+```
+
+since we need to set a global environment variable that works for `redis_om`.
```

Comment:
This is useful, maybe it could a new FAQ entry? What is the symptom of doing this wrong?

Reply:
if we set this, I came across authorization error.

Reply:
Cool! Would you mind moving your answer under a new question like "Why do I encounter authorization error '<input the concrete error message>' after setting `REDIS_OM_URL` in Python `os.environ`"?

Reply:
yes, will do later.

Reply:
UPDATED!